powercmd/command.py

```diff
@@ -74,10 +74,16 @@ def short_description(self) -> str:
             return self.description.strip().split('\n', maxsplit=1)[0]
         return None
 
+    def _print_param(self, param):
+        if self.parameters[param].default is inspect.Parameter.empty:
+            return str(param)
+        else:
+            return str(param) + '*'
```

Comment:
I'm aware that's a subjective thing, but I don't remember seeing `*` used to mean "optional" as in "zero or one", which is the intent here. When seeing it for the first time, I'd most probably assume it means that the parameter can be repeated 0+ times (matching the meaning of `*` in regexes).

What do you think about one of those alternatives?
- `param?` (regex-like)
- `[param]` (`man`-like)
- `'param=' + repr(default)` (python-like)